pkg/validation/validate.go

```diff
@@ -128,3 +129,13 @@ var LineLengthHist = promauto.NewHistogram(prometheus.HistogramOpts{
 	Help:      "The total number of bytes per line.",
 	Buckets:   prometheus.ExponentialBuckets(1, 8, 8), // 1B -> 16MB
 })
+
+// DuplicateLogEntries is a metric of the total discarded duplicate bytes, by tenant.
+var DuplicateLogEntries = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Namespace: constants.Loki,
+		Name:      "duplicate_log_entries_total",
```

Comment:
I believe this metric is a byte counter not a line counter so this should probably be `duplicate_log_bytes_total`